api_tests/registrations/views/test_registration_list.py

```diff
@@ -1418,47 +1407,40 @@ def test_assert_file_validity_on_registration_metadata(
         version.metadata['sha256'] = sha256
         version.save()
 
-        prereg_schema = RegistrationSchema.objects.get(
-            name='Prereg Challenge',
-            schema_version=SCHEMA_VERSION)
+        open_ended_schema = RegistrationSchema.objects.get(
+            name='Open-Ended Registration',
```

Comment:
nit: Other tests in this file use the OSF Preregistration schema. Is there a specific reason you chose then open ended registration schema?

Reply:
The open ended registration schema is the easiest to write tests for while sufficiently meeting the purpose of the tests. Using a more complicated schema makes these tests more likely to fail for reasons unrelated to their purpose.

Reply:
👌  Sounds good! Thanks!